bound/bound.go

```diff
@@ -20,6 +20,13 @@ func (b Bound) IsValid() bool {
 	return b.Min <= b.Max
 }
 
+// Jaccard returns a jaccard index between bounds
```

Comment:
Final full stop.

I would give a caveat here that the JAccard result is only correct for sets of bounds that have a contiguous union.

Reply:
Do you think this should be called JaccardIndex since there is also a well-defined JaccardDistance?

Reply:
@btracey 
Okay, I will update it into JaccardDistance :)

Reply:
`JaccardIndex`

Reply:
@btracey @kortschak Oh.. I mistranslated... I 've now understood what you are saying!